data_structures/src/radon_error.rs

```diff
@@ -52,11 +52,17 @@ pub enum RadonErrors {
     TallyExecution = 0x53,
     /// Invalid reveal serialization (malformed reveals are converted to this value)
     MalformedReveal = 0x60,
+    /// Invalid data request serialization (malformed data requests are converted to this value)
+    MalformedRequest = 0x61,
```

Comment:
I wonder whether malformation is the only reason for nonacceptance of a request. For example, an excessive number of witnesses could also be a reason for nonacceptance.

Generally speaking, it feels to me like we should be able to communicate the Ethereum client any reason why a request could be rejected by the Witnet network, either by specifying every case, or lumping all of them together into something generic.

Reply:
I think that errors that only belong in inter-client communication (ethereum<>relayer<>witnet-rust) but cannot exist in tallies should all belong in a new category, for which we can use a dedicated prefix, e.g. `0xE...` so they are not confused with proper witnessing errors.